new-frontiers-14/frontier-station-14
# Added rubber ammo to NFSD TechFab

## About the PR


Moved nonlethal recipes from dynamic to static for SecurityTechFab inherited by NfsdTechFab

## Why / Balance

Less-lethal ammunition is available from round start via vending machines already, but the NFSD techfab only has lethal ammunition by default.

Allowing the techfab to produce less-lethal rounds in the field will help encourage de-escalation

Typically there are only 1-2 nfsd fabricators in play at any given time, so impact will be limited.

## How to test

Create NFSD techfab from flatpack or spawn menu
Verify rubber recipes are available by default
Link with research server to ensure fabricator does not break if linked to R&D

## Media

<img width="432" alt="NFSD Techfab Rubber" src="https://github.com/user-attachments/assets/4346e566-f785-4012-858a-87f3c873d472">


## Requirements



## Breaking changes

N/A

**Changelog**


:cl:
- tweak: Added less-lethal ammunition to NFSD techfab by default, with additional NFSD items.
